.github/workflows/tests.yml

```diff
@@ -70,3 +70,68 @@ jobs:
           name: failure-screenshots-trace
           path: test-results
           retention-days: 10
+
+  deploy_report:
+    name: Deploy report to GH Pages
+    needs: [api_tests, e2e_tests]
+    # if: always() && github.ref == 'refs/heads/main'
```

Comment:
Let's enable this only for `main` jobs, this will be reverted before merging.

Reply:
got it 🎯